javascript/testing_javascript/more_testing.md

```diff
@@ -64,7 +64,7 @@ If we had written this program with TDD it is very likely that it would have loo
 ### Assignment 
 
 <div class="lesson-content__panel" markdown="1">
-1. If you haven't already, watch the 'mocking' videos from [this series](https://www.youtube.com/watch?v=3PjdxjWK0F0).
+1. Check out the mocking portion from [this crash course](https://www.youtube.com/watch?v=ajiAl5UNzBU&ab_channel=LaithAcademy).
```

Comment:
I think we could add a link to the exact section instead of the whole video: https://www.youtube.com/watch?v=ajiAl5UNzBU&t=3024s

Reply:
i am new to git hub and just wanted to give some meanful contributions. if you want me to do something about this pull request feel free to tell me.

Reply:
@MrSaadMasood you should have a button that will accept their suggested changes.  Just click on that and all should be good.